test/functional-tests/pageObjects/ccd-components/fields/ccdFixedList.js

```diff
@@ -26,6 +26,25 @@ class CcdFixedList {
         this.label = await this._getLabel();
     }
 
+      /**
+     * Check if field is ready to type
+     * @returns true or false
+     */
+    async isFieldInputReady(optionsTextValues){
```

Comment:
lets change function name to something more related to a fixedlist/dropdown being ready and update the doc comment accordingly